peps/pep-0738.rst

```diff
@@ -3,12 +3,14 @@ Title: Adding Android as a supported platform
 Author: Malcolm Smith <smith@chaquo.com>
 Sponsor: Petr Viktorin <encukou@gmail.com>
 Discussions-To: https://discuss.python.org/t/pep-738-adding-android-as-a-supported-platform/40975
-Status: Accepted
+Status: Final
 Type: Standards Track
 Created: 12-Dec-2023
 Python-Version: 3.13
 Resolution: https://discuss.python.org/t/pep-738-adding-android-as-a-supported-platform/40975/23
 
+.. canonical-doc:: :ref:`py3.13:using-android`
```

Comment:
The Python 3.13 docs need to be added to `intersphinx_mapping` in `conf.py` for this to work—should be a quick one-line change, but I'm happy to commit it directly to this branch, if you'd like.

Reply:
Python 3.13 is already in that list – the reason the build is failing is that https://github.com/python/cpython/pull/124259 hasn't been merged yet.

But even once it has been merged, I think this link wouldn't be valid until the next 3.13 release. So I've changed it to target 3.14 instead.

Reply:
> I think this link wouldn't be valid until the next 3.13 release

It will be valid from when the 3.13 documentation is rebuilt, so targeting 3.13 is probably the right thing to do (unless this can wait until after 3.13.0 is released, as linking to /3/ would be nicer and more 'evergreen').

A

Reply:
> Python 3.13 is already in that list – the reason the build is failing is that https://github.com/python/cpython/pull/124259 hasn't been merged yet.

Oops, my bad—I'd been looking at an old version of `conf.py` in my local tree and just assumed that was the issue instead of carefully investigating the error. Thanks for the catch!

> unless this can wait until after 3.13.0 is released, as linking to /3/ would be nicer and more 'evergreen'

That would be ideal, considering otherwise someone has to remember to go back and update it, and its not _that_ long a wait now...in the past, as an informal convention I often waited to mark PEPs Final until the actual release of the version the change was contained in, if tied to the Python core release schedule.

Reply:
@mhsmith it might be better to split the PRs -- we can update PEP 11 first to make Android support official, and then later mark this PEP as final. I'd like to include Android in *What's New in Python 3.13* (https://github.com/python/cpython/pull/124259) if possible.

A

Reply:
OK, I have split the PEP 11 changes into #3989, and changed the canonical-doc link to the `/3/` URL.